tests/spread/plugins/v2/make/run/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Build and run a basic make snap
```

Comment:
it might make more sense to allow the existing tests to be extended by adding the 20.04 target? otherwise we're going to have a lot of duplicate tests...  maybe use a symlink for things that can/should be shared?

Reply:
yeah, I think I want to move all the snap dirs to a common location at the root of the spread tests.

Reply:
I took a quick look at merging everything, that would require fine tuning all the individual tasks for things not supported in core20 if they do not live in their own suite.